controllers/osccluster_loadbalancer_controller_unit_test.go

```diff
@@ -511,11 +516,28 @@ func TestReconcileLoadBalancer(t *testing.T) {
 			expSecurityGroupFound:         true,
 			expCreateLoadBalancerFound:    true,
 			expConfigureLoadBalancerFound: true,
+			expDeleteOutboudSgRule:        true,
+			expDeleteOutboudSgRuleErr:     nil,
 			expDescribeLoadBalancerErr:    nil,
 			expCreateLoadBalancerErr:      nil,
 			expConfigureLoadBalancerErr:   nil,
 			expReconcileLoadBalancerErr:   nil,
 		},
+		{
+			name:                          "failed to empty outboud Sg for loadBalancer",
```

Comment:
To delete default outbound ?

Reply:
(empty is an adjectif not a verb https://dictionary.cambridge.org/fr/dictionnaire/anglais/empty)

Reply:
it is also a verb, but i can also use delete if you want